src/event.js

```diff
@@ -187,17 +188,18 @@ jQuery.event = {
 
 				// Only use addEventListener if the special events handler returns false
 				if ( !special.setup ||
-					special.setup.call( elem, data, namespaces, eventHandle ) === false ) {
+					special.setup.call( elem, data, namespaces, handleObj ) === false ) {
 
 					if ( elem.addEventListener ) {
-						elem.addEventListener( type, eventHandle );
+						elem.addEventListener( type, handleObj );
```

Comment:
This is still guarded by:
```js
if ( !( handlers = events[ type ] ) ) {
```
making it one `addEventListener` per event type... Or am I reading it wrong?